bin/kibana-plugin

```diff
@@ -21,4 +21,4 @@ if [ ! -x "$NODE" ]; then
   exit 1
 fi
 
-exec "${NODE}" $NODE_OPTIONS "${DIR}/src/cli_plugin" ${@}
+exec "${NODE}" $NODE_OPTIONS --no-warnings "${DIR}/src/cli_plugin" ${@}
```

Comment:
The kibana-plugin is currently not displaying the warnings at all, currently investigating the best way to determine if "we're in production"